### DISCUSSION TITLE
Welcome to AYUSH-Portal-Frontend Discussions!

### DISCUSSION THREAD

**Discussion by BeeBombshell** [BeeBombshell/AYUSH-Portal-Frontend#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BhavyaT-135** [comment]
Hi, I'm Bhavya Tewari. I've been working on the Login and SignUp pages for the Portal. Looking forward to committing the changes and checking if everything works fine! 😃

**Reply by BeeBombshell** [reply]
Hi @BhavyaT-135! Sure, I added you as a collaborator! Please go ahead with the changes! 😄🚀
